fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    /* your code here */
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(response => response.json())
+      .then(data => this.setState({
+        users: data
+      }));
+    this.setState({
+      isLoading: false
+    });
```

Comment:
The `isLoading` should be set at the same time as the `user` data, not in a separate call to `setState`
```JAVASCRIPT
fetch('https://jsonplaceholder.typicode.com/users')
  .then(response => response.json())
  .then(data => this.setState({
    users: data,
    isLoading: false
  }));
```